src/redist/targets/packaging/windows/clisdk/VS.Redist.Common.NetCore.Toolset.nuspec

```diff
@@ -1,15 +1,15 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <package xmlns="http://schemas.microsoft.com/packaging/2011/08/nuspec.xsd">
     <metadata>
-        <id>VS.Redist.Common.NetCore.Toolset.$ARCH$</id>
+        <id>VS.Redist.Common.NetCore.Toolset.$DATA$</id>
```

Comment:
What is the format of this data? I feel like we went from a very descriptive name to a very generic one. I would prefer a name that is more indicative of the purpose of this variable.

Reply:
I can understand that preference - changing it back.